tests/test_simpleion.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#  
+#     http://www.apache.org/licenses/LICENSE-2.0
+#  
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import amazon.ion.simpleion as ion
+from .util import hex_string
+
+
+def test_simpleion_invalid_no_params():
+    try:
+        ion.loads('blah').ion_hash()
+        raise Exception("Expected an exception to be raised")
```

Comment:
pytest has a context manager called `raises` that you may find is a cleaner way to test for expected exceptions. See https://github.com/amzn/ion-python/blob/master/tests/test_vectors.py#L208 for an example.

Reply:
ok, thanks.  feeling cooler already :)